content/home/examples/a-stateful-component.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: A Stateful Component
+title: Komponent ze stanem
 order: 1
 domid: timer-example
 ---
 
-In addition to taking input data (accessed via `this.props`), a component can maintain internal state data (accessed via `this.state`). When a component's state data changes, the rendered markup will be updated by re-invoking `render()`.
+Poza danymi wejściowymi do komponentu (dostępnymi w zmiennej `this.props`), komponent może zarządzać swoim wewnętrznym stanem (dostępnym pod `this.state`).  Przy każdej zmianie stanu komponentu następuje ponownie wywołanie metody `render()`, co skutkuje zaktualizowaniem wyrenderowanej struktury.
```

Comment:
`w this.props`, ale `pod this.state`? Troche brak spójności.

Reply:
Masz rację, przydałoby się zachować tu konsekwencję. "pod" to był skrót myślowy od "pod nazwą", ale to tak czy inaczej nie pasuje do tego kontekstu. Myślę, że "obiekt" lub "właściwość" będą tu lepiej pasować.

Reply:
Ewentualnie można pominąć to słowo przed i zostawić samo "dostępnym w `this.props`" i "dostępnym w `this.state`".

Reply:
A ja nie widzę tu najmniejszego problemu. Co więcej wydaje mi się, że wersja początkowa brzmi lepiej.